event.py

```diff
@@ -0,0 +1,26 @@
+from sqlalchemy import Column, DateTime, Text, Integer
+from dbsetting import Base, ENGINE
+
+class Event(Base):
+    """
+    Event model
+    """
+
+    __tablename__ = "event"
+    __table_args__=({"mysql_engine": "InnoDB"})
+    time = Column("time", DateTime, nullable=False, primary_key=True)
+    area = Column("area", Text, nullable=False)
+    cause = Column("cause", Text, nullable=False)
+    direction = Column("direction", Text, nullable=False, primary_key=True)
+    section = Column("section", Text, nullable=False, primary_key=True)
+    route = Column("route", Text, nullable=False)
+    detail = Column("detail", Text, nullable=False)
+    length = Column("length", Integer, nullable=True)
+    coordinates = Column("coordinates", Text, nullable=True)
```

Comment:
These should be defined under the function `def init(self)`, as a part of the initiation of the class

Reply:
Ok, but what is the downside of not using `def init(self)`? I googled tutorials and they don't `__init__` in the class.

Reply:
class object will run everything defined in `__init__` when the var which uses the class get declared. So anything you need for a class to be pre-defined before running any functionalities, I recommend to define it in `__init__`